.gitignore

```diff
@@ -1,4 +1,145 @@
-*.pyc
-*.egg-info
-__pycache__
-/dist
+# Byte-compiled / optimized / DLL files
```

Comment:
I propose to remove `.gitignore` changes from this pull request. Please create new pull request if you think `.gitignore` has to be updated.

Reply:
Sounds good, will do